iceoryx_binding_c/README.md

```diff
@@ -54,10 +70,25 @@ enum iox_Color {
     Color_BLUE,
 };
 
-CLASS MyTransmitterHandle* iox_sub_create(const char * name);
-void iox_sub_destroy(CLASS MyTransmitterHandle * self);
-bool iox_sub_receive(void * const data);
-bool iox_sub_send(void * const data);
-void iox_sub_all_hail_hypnotoad();
+
+typedef struct Subscriber * sub_t;
+
+sub_t iox_sub_create(const char * name);
+void iox_sub_destroy(sub_t const self);
+bool iox_sub_receive(sub_t const self, void * const data);
+bool iox_sub_send(sub_t const self, void * const data);
+void iox_sub_all_hail_hypnotoad(sub_t const self);
+
+
+struct iox_other_class_storage_t_ {
+    uint64_t do_not_touch_me[8]; // lets assume 8 * 8 is the size of MyOtherClass
```

Comment:
why not uint8_t, i.e. we count bytes not blocks of 8 bytes? (or unsigned char or similar 1 byte types)

Reply:
@MatthiasKillat we require a alignment of 8 bytes and this can only be achieved if we are using a member which is aligned to 8 bytes.

Reply:
Is the alignment requirement mentioned somewhere? Implicitly ensuring it by using uint64 might be dangerous?!

Reply:
It is tested in the unit tests.
We already have unit tests which verify that this constructs have the correct alignment and the correct size, see https://github.com/elfenpiff/iceoryx/blob/iox-%23252-c-binding-wait-set/iceoryx_binding_c/test/integrationtests/test_types.cpp